### DISCUSSION TITLE
Renaming Cues

### DISCUSSION THREAD

**Discussion by PhilippKuhn** [FrancescoCeruti/linux-show-player#309] [Q&A] [answered]
Hi,

I really appreciate the idea of the program, because there is nothing similar out there and it can be really useful for Linux-Users. I wonder if it's possible to rename the Cues? For example rename The general "Command Cue" in "StartClipAliceInWonderland"? It would be helpful to create a custom Page for a specific piece.

**Comment by FrancescoCeruti** [comment]
Yes, you can change the name: https://linux-show-player-users.readthedocs.io/en/latest/cues/index.html#cues-options

**Comment by PhilippKuhn** [comment]
Hi Francesco, ok. I found that part but in the GUI it don't allows me to change something.

**Reply by FrancescoCeruti** [reply]
I've taken a short video:

[Video del 2024-02-10 19-53-30.webm](https://github.com/FrancescoCeruti/linux-show-player/assets/5596673/f04d375a-8565-40ef-a677-2badec98f914)

**Comment by PhilippKuhn** [comment]
Hi Francesco,
thanks for the Video. With my Installation it's not like this. I already installed with flatpak. 


https://github.com/FrancescoCeruti/linux-show-player/assets/65501911/976e9d07-32ce-45db-884b-843ba7fcc7ff



Do I need another version?

**Reply by FrancescoCeruti** [reply]
Something is definitely wrong, can you post the output of `flatpak run org.linuxshowplayer.LinuxShowPlayer --log=debug`?

**Reply by FrancescoCeruti** [reply]
Let me also know which distro (and it's version) you're using

**Comment by PhilippKuhn** [comment]
Hi, I am using Ubuntu 20.04
[output.txt](https://github.com/FrancescoCeruti/linux-show-player/files/14233687/output.txt)

**Reply by s0600204** [reply, marked as answer]
Would I be right in thinking you're using a 4K screen with some form of UI scaling enabled? If so, does the problem still happen if you change the scaling off?

**Reply by PhilippKuhn** [reply]
Hi Francesco, you're right, that solved the problem. I found another Issue, when I install the Player with Flatpak, the command cue is not working. Just the one I get via sudo apt-install linux-show-player. It is ok. to work with the older one but Usually I would prefer to have it work like a normal application.. Maybe you have an idea why it is not working?

**Reply by FrancescoCeruti** [reply]
It wasn't me :laughing: but I'm glad you solved the problem :) The Flatpak is sandboxed, it need some special treatment, see #310
